examples/rgbpp/env.ts

```diff
@@ -48,8 +48,77 @@ export const BTC_SERVICE_ORIGIN = process.env.VITE_BTC_SERVICE_ORIGIN!;
 // - P2WPKH: https://github.com/bitcoin/bips/blob/master/bip-0141.mediawiki#p2wpkh
 // - P2TR: https://github.com/bitcoin/bips/blob/master/bip-0341.mediawiki
 const addressType = process.env.BTC_ADDRESS_TYPE === 'P2TR' ? AddressType.P2TR : AddressType.P2WPKH;
-const networkType = isMainnet ? NetworkType.MAINNET : NetworkType.TESTNET;
+export const networkType = isMainnet ? NetworkType.MAINNET : NetworkType.TESTNET;
 export const btcAccount = createBtcAccount(BTC_PRIVATE_KEY, addressType, networkType);
 
 export const btcService = BtcAssetsApi.fromToken(BTC_SERVICE_URL, BTC_SERVICE_TOKEN, BTC_SERVICE_ORIGIN);
 export const btcDataSource = new DataSource(btcService, networkType);
+
+export const initOfflineCkbCollector = async (
+  queries: {
+    lock?: CKBComponents.Script;
+    type?: CKBComponents.Script;
+    isDataMustBeEmpty?: boolean;
+    outputCapacityRange?: Hex[];
+    withEmptyType?: boolean;
+  }[],
+) => {
+  const cells = (
+    await Promise.all(
+      queries.map(async (query) => {
+        let cells = await collector.getCells(query);
+        if (!cells || cells.length === 0) {
+          throw new Error(`No cells found for query: ${JSON.stringify(query)}`);
+        }
+        if (query.withEmptyType) {
+          cells = cells.filter((cell) => !cell.output.type);
+        }
+        if (cells.length === 0) {
+          throw new Error(`No empty cells found for query: ${JSON.stringify(query)} with empty type`);
+        }
+        return Promise.all(
+          cells.map(async (cell) => {
+            const liveCell = await collector.getLiveCell(cell.outPoint);
+            return {
```

Comment:
I think `liveCell` is enough for the offline collector and the `status` and `outputDataHash` are redundent.

Reply:
Agreed. I misunderstood the functionality of the RPC methods `get_cells` and `get_live_cell`. These 2 fields were removed in commit 0ff259d86f552ecbf0eb28577be486fdc14fea7a.